### DISCUSSION TITLE
Does Zed support Unicode chars in its terminal?

### DISCUSSION THREAD

**Discussion by milosgajdos** [zed-industries/zed#10106] [Feature Requests]
I use [starship](https://starship.rs/) for my terminal prompt and it inserts some unicode characters which work like charm in alacritty/iterm2 but break miserably in zed.

Alacritty:
<img width="383" alt="Screenshot 2024-04-03 at 08 52 58" src="https://github.com/zed-industries/zed/assets/1392526/4a6b5379-0160-4e8d-874c-bfd4cb71df56">

Zed terminal:
<img width="462" alt="Screenshot 2024-04-03 at 08 53 29" src="https://github.com/zed-industries/zed/assets/1392526/bc733e51-8607-47f4-a7f2-5779c435e237">

See the `[?]` next to the `main` branch name. Curious if I need to import something or change some config?

Surprisingly enough, the 🦀 emoji renders like a charm.

I couldnt find anything in the GH issues but I found [this](https://github.com/zed-industries/zed/discussions/9150) so I'm curious how to make this work if possible.

**Comment by d1y** [comment]
You need to use nerd-font
```jsonc
"terminal": {
	"font_family": "JetBrainsMono Nerd Font Mono", // I am using JetBrainsMono
}
```
<img width="445" alt="image" src="https://github.com/zed-industries/zed/assets/45585937/c2c77a2f-44e5-4b6d-bda3-8896c00f7081">

---------------------

Install nerd-font: https://github.com/ryanoasis/nerd-fonts#option-2-homebrew-fonts

```bash
brew tap homebrew/cask-fonts

# And. Then
❯ brew search nerd-font
# Just install a favorite nerd-font(patch)
==> Casks
homebrew/cask-fonts/font-0xproto-nerd-font                    homebrew/cask-fonts/font-fira-mono-nerd-font                  homebrew/cask-fonts/font-monaspace-nerd-font
homebrew/cask-fonts/font-3270-nerd-font                       homebrew/cask-fonts/font-geist-mono-nerd-font                 homebrew/cask-fonts/font-monocraft-nerd-font
homebrew/cask-fonts/font-agave-nerd-font                      homebrew/cask-fonts/font-go-mono-nerd-font ✔                  homebrew/cask-fonts/font-monofur-nerd-font
homebrew/cask-fonts/font-anonymice-nerd-font                  homebrew/cask-fonts/font-gohufont-nerd-font                   homebrew/cask-fonts/font-monoid-nerd-font
homebrew/cask-fonts/font-arimo-nerd-font                      homebrew/cask-fonts/font-hack-nerd-font                       homebrew/cask-fonts/font-mononoki-nerd-font
homebrew/cask-fonts/font-aurulent-sans-mono-nerd-font         homebrew/cask-fonts/font-hasklug-nerd-font                    homebrew/cask-fonts/font-mplus-nerd-font
homebrew/cask-fonts/font-bigblue-terminal-nerd-font           homebrew/cask-fonts/font-heavy-data-nerd-font                 homebrew/cask-fonts/font-noto-nerd-font
homebrew/cask-fonts/font-bitstream-vera-sans-mono-nerd-font   homebrew/cask-fonts/font-hurmit-nerd-font                     homebrew/cask-fonts/font-open-dyslexic-nerd-font
homebrew/cask-fonts/font-blex-mono-nerd-font                  homebrew/cask-fonts/font-im-writing-nerd-font                 homebrew/cask-fonts/font-overpass-nerd-font

# brew install font-hack-nerd-font
```

**Comment by milosgajdos** [comment]
That didn't do the trick, but I presume I need to install these fonts first before you suggestion can work 🤔

**Reply by milosgajdos** [reply]
Actually, the docs claim that 
> The name of any font family installed on the user's system

Which means that the font I used in Alacritty is not the Jetbrains Mono you are suggesting. I had a look and I'm using "MesloLGS NF" which is indeed installed on my OS, ~but I dont seem to be able to use it in Zed~ 😬

Specifying it in the config as per @d1y suggestion worked like charm! ❤️ 

~Is there a way how I can somehow "import" the fonts or something? I couldnt find anythiing in the docs so I think must be overloooking somethiing~
